projects/packages/woocommerce-analytics/changelog/tweak-remov-prefix-woocommerceanalytics-clickhouse

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Remove prefix _woocommerceanalytics in ClickHouse events
```

Comment:
I guess we don't need this change log as we are keeping the prefix for now. In the next PR maybe we can merge all change logs into one or two to make it more specifically before merging into trunk.